doc/nrf/ug_bootloader.rst

```diff
@@ -47,19 +47,22 @@ The following image shows an abstract representation of the memory layout, assum
 
 For detailed information about the memory layout, see the partition configuration in the :file:`<build folder>/partitions.yml` file or run ``ninja partition_manager_report``.
 
+.. note::
+   To see above information, multi image need to be used. See :ref:`partition_manager` and :ref:`ug_multi_image` for details.
```

Comment:
```suggestion
   The Partition Manager must be enabled for the ``partition_manager_report`` to be available. See :ref:`partition_manager` and :ref:`ug_multi_image` for details.
```

Reply:
Would you accept below instead?
The Partition Manager must be enabled for the ``partitions.yml`` and ``partition_manager_report`` to be available. See :ref:`partition_manager` and :ref:`ug_multi_image` for details.

Reply:
I think its important to highlight that one is a file and one is a ninja target.

```
:file:`partitions.yml` and ``partition_manager_report`` target
```

Reply:
I agree, fixed.